src/main/java/ThomasShelby.java

```diff
@@ -0,0 +1,105 @@
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.io.File;
+import tasks.*;
+
+public class ThomasShelby {
+    static ArrayList<Task> taskManager = new ArrayList<>();
+
+    private static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskManager.size(); i++) {
+            System.out.println((i + 1) + ". "
+                    + taskManager.get(i));
+        }
+    }
+
+    private static void addToDo(String[] cmdSplit) {
+        ToDo newToDo = new ToDo(cmdSplit[1]);
+        taskManager.add(newToDo);
+        System.out.println("Don't sleep on it.\n" + "added: " + newToDo);
+    }
+
+    private static void addDeadline(String[] cmdSplit) {
+        String[] taskAndDeadline = cmdSplit[1].split("/", 2);
```

Comment:
Java Coding Standard: Plural form should be used on names representing a collection of objects.

Can consider renaming to `splitTaskParameters` across this and other occurences.